src/main/java/com/iexec/worker/compute/pre/PreComputeAppRunner.java

```diff
@@ -40,7 +44,7 @@ public int start() {
         ReplicateStatusCause exitCause = ReplicateStatusCause.PRE_COMPUTE_FAILED_UNKNOWN_ISSUE;
         String chainTaskId = "";
         try {
-            chainTaskId = PreComputeArgs.getEnvVarOrThrow(IEXEC_TASK_ID);
+            chainTaskId = EnvUtils.getEnvVarOrThrow(IEXEC_TASK_ID, PreComputeArgs.buildReplicateCauseIfMissing(IEXEC_TASK_ID.name()));
```

Comment:
Wouldn't it be simpler to just do this ?
```suggestion
            chainTaskId = EnvUtils.getEnvVarOrThrow(IEXEC_TASK_ID, ReplicateCause.PRE_COMPUTE_TASK_ID_MISSING);
```

Reply:
Resolved in [9fda284](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/pull/99/commits/9fda2842520ca3b5a79d102c829ebb0a62542960)